api/controllers/expenseControllers.js

```diff
@@ -0,0 +1,27 @@
+const Expense = require('../models/expense');
+
+const expenseCreate = (req, res) => {
+  const { amount, description, budget, category } =  req.body;
+  const newExpense = new Expense({ amount, description, budget, category });
+  newExpense.save(newExpense, (err, savedExpense) => {
+    if (err) {
+      res.status(500).json(err);
+      return;
+    }
+    res.json(savedExpense);
+  })
+};
+
+const getAllexpenses = (req, res) => {
+	Expense.find({})
+		.then(expenses => {
```

Comment:
So this is where you'd want to use the populate method to join the information from the data sets and return that info out to the user. I recommend you give this a try!